lib/common/shared/src/main/scala/net/wiringbits/common/models/Email.scala

```diff
@@ -20,4 +21,21 @@ object Email extends WrappedString.Companion[Email] {
   }
 
   override def trusted(string: String): Email = new Email(string)
+
+  implicit val emailColumn: Column[Email] = Column.nonNull[Email] { (value, _) =>
```

Comment:
If I understand correctly, the reason to do this is because Typo picks these models and expects the `Column` definition to be available.

Let's try adding all these definitions into the typo project, we can use the scala-cli script to import these into each generated source.